src/app/shared/components/confirmation-dialog/confirmation-dialog.component.ts

```diff
@@ -8,6 +8,11 @@ export enum DialogSeverity {
   DANGER = 'DANGER',
 }
 
+export enum DialogColorTheme {
```

Comment:
`color` seems to be implied by `theme`, so you could just name it `DialogTheme` :)

Reply:
I would say that either `DialogTheme` or `DialogColorScheme`. Personally I am favoring `DialogColorScheme` (or `ColorScheme`), since theme often include other stylistic attributes than color.

Reply:
Rename